docs/docs/rest-api/public/api/v2/types/mesosContainer.raml

```diff
@@ -0,0 +1,43 @@
+#%RAML 1.0 DataType
+type: object
+description: Mesos Container
+properties:
+  name:
+    type: !include name.raml
+    description: The name of this container
+  command?:
+    type: !include mesosCommand.raml
+    description: command line executed by the default shell, not parsed by marathon
+  resources:
+    type: !include resources.raml
+    description: The resources to allocate to the container.
+  endpoints?:
+    type: array
+    items: !include endpoint.raml
+    description: The ports needed to run this container
+  image?:
+    type: !include image.raml
+    description: The filesystem image to populate the container with
+  environment?:
+    type: array
+    items: !include environmentVariable.raml
+    description:|
+      Environment variables to set, defined as key/value pairs.
+      Variables defined here override those specified in the pod definition.
+  user?:
+    type: string
+    description:|
+      The OS user to use to run the tasks on the agent.
+      Values here override a "user" value specified in the pod definition.
+  healthCheck?:
+    type: !include healthCheck.raml
+    description: All healthchecks to perform on the container
+  volumeMounts?:
+    type: array
+    items: !include volumeMount.raml
+    description: All volume mounts
+  artifacts?:
```

Comment:
The initial v3 api doc specified artifacts on pod level (= shared artifacts) **and** on container level.
Would artifacts on pod level only be sufficient for the MVP? Just wondering because this api spec doesn't allow artifacts on pod level.